src/components/PoweredByGoogleLogo/index.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import styled from "styled-components"
+
+const GoogleWrapper = styled.div`
+  text-align: end;
+  padding: 0.3125rem;
+`
+
+class PoweredByGoogleLogo extends React.Component {
+  render() {
+    return (
+      <GoogleWrapper>
+        <img
+          alt={"Powered By Google"}
+          src={
+            "../../assets/images/powered_by_google/desktop/powered_by_google_on_white.png"
```

Comment:
Do we need to include all the other version of the asset if we're just using this one?

Reply:
I figured it might be better to include the rest of google's supplied logos if there's a need to extend this for future developers.